arms_up: {arms: b, frame_left: torso_lift_link, frame_right: torso_lift_link, grip_left: false,
  grip_right: false, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.065610807345472519,
  pitch_right: -0.0057656540516995711, roll_left: -3.1379996547187234, roll_right: 2.9315085315142437,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.34044899044259058,
  x_right: 0.40193335037874661, y_left: 0.58311362158899582, y_right: -0.45875303609237239,
  yaw_left: -0.13377688161257423, yaw_right: 0.25185212176193561, z_left: 0.34823149207144399,
  z_right: 0.37168224442988773}
arms_up_grip: {arms: b, frame_left: torso_lift_link, frame_right: torso_lift_link,
  grip_left: true, grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.065610807345472519,
  pitch_right: -0.0057656540516995711, roll_left: -3.1379996547187234, roll_right: 2.9315085315142437,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.34044899044259058,
  x_right: 0.40193335037874661, y_left: 0.58311362158899582, y_right: -0.45875303609237239,
  yaw_left: -0.13377688161257423, yaw_right: 0.25185212176193561, z_left: 0.34823149207144399,
  z_right: 0.37168224442988773}
examine_height: {arms: n, frame_left: base_laser_link, frame_right: base_laser_link,
  grip_left: false, grip_right: false, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0,
  pitch_right: 0, roll_left: 0, roll_right: 0, set_head_angle: false, set_torso_height: true,
  torso_height: 0.30008862501536443, x_left: 0, x_right: 0, y_left: 0, y_right: 0,
  yaw_left: 0, yaw_right: 0, z_left: 0, z_right: 0}
flip_mid: {arms: b, frame_left: base_footprint, frame_right: base_footprint, grip_left: true,
  grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.31097795766529068,
  pitch_right: 0.0056368627666804659, roll_left: 1.5043369595456222, roll_right: 1.8651175406502452,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.74027987314594079,
  x_right: 0.76163323680134687, y_left: 0.026239011668581336, y_right: -0.0085332603513146033,
  yaw_left: -1.2668260651351544, yaw_right: 1.564971697059675, z_left: 1.1995777652313877,
  z_right: 1.1965289237406633}
flip_up: {arms: b, frame_left: base_footprint, frame_right: base_footprint, grip_left: true,
  grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.23391513647831552,
  pitch_right: 0.021985250545353354, roll_left: 1.6015903578056487, roll_right: 1.7892756377487404,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.73743997472763723,
  x_right: 0.753549624210053, y_left: 0.031844418345506054, y_right: -0.0084117598392120241,
  yaw_left: -1.2091979635196077, yaw_right: 1.5142851519691998, z_left: 1.2739608453638898,
  z_right: 1.2766415423848081}
look_down: {arms: n, frame_left: torso_lift_link, frame_right: torso_lift_link, grip_left: false,
  grip_right: false, head_pan: 0.0, head_tilt: 1.0, pitch_left: 0, pitch_right: 0,
  roll_left: 0, roll_right: 0, set_head_angle: true, set_torso_height: false, torso_height: 0.0,
  x_left: 0, x_right: 0, y_left: 0, y_right: 0, yaw_left: 0, yaw_right: 0, z_left: 0,
  z_right: 0}
look_down_more: {arms: n, frame_left: torso_lift_link, frame_right: torso_lift_link,
  grip_left: false, grip_right: false, head_pan: 0.0, head_tilt: 1.1499999999999999,
  pitch_left: 0, pitch_right: 0, roll_left: 0, roll_right: 0, set_head_angle: true,
  set_torso_height: false, torso_height: 0.0, x_left: 0, x_right: 0, y_left: 0, y_right: 0,
  yaw_left: 0, yaw_right: 0, z_left: 0, z_right: 0}
look_up: {arms: n, frame_left: torso_lift_link, frame_right: torso_lift_link, grip_left: false,
  grip_right: false, head_pan: 0.0, head_tilt: 0.5, pitch_left: 0, pitch_right: 0,
  roll_left: 0, roll_right: 0, set_head_angle: true, set_torso_height: false, torso_height: 0.0,
  x_left: 0, x_right: 0, y_left: 0, y_right: 0, yaw_left: 0, yaw_right: 0, z_left: 0,
  z_right: 0}
put_in_bucket: {arms: r, frame_left: base_footprint, frame_right: base_footprint,
  grip_left: false, grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0,
  pitch_right: 1.3143479998568841, roll_left: 0, roll_right: 2.5520096156612961, set_head_angle: false,
  set_torso_height: false, torso_height: 0.0, x_left: 0, x_right: 0.17334518082712674,
  y_left: 0, y_right: -0.98006875308991614, yaw_left: 0, yaw_right: -1.6421358933004984,
  z_left: 0, z_right: 1.0532362634345021}
smooth1: {arms: b, frame_left: base_footprint, frame_right: base_footprint, grip_left: true,
  grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.56556594694061757,
  pitch_right: 0.72669341770408058, roll_left: -0.087787302199123937, roll_right: 0.09554088415144367,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.42420984173849958,
  x_right: 0.42458447317195553, y_left: 0.010410924364052757, y_right: -0.013768195966079032,
  yaw_left: -1.5751855731635112, yaw_right: 1.7229936060677593, z_left: 0.83916373097600139,
  z_right: 0.83438215529489701}
smooth2: {arms: b, frame_left: base_footprint, frame_right: base_footprint, grip_left: true,
  grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.6522792259974558,
  pitch_right: 0.84675969649792215, roll_left: -0.09230796612560177, roll_right: -0.043345564941896245,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.40480871621732457,
  x_right: 0.418949862304975, y_left: 0.011666467568457998, y_right: -0.002777822952243824,
  yaw_left: -1.6005368882882158, yaw_right: 1.7120306179655786, z_left: 0.78661443071090564,
  z_right: 0.78035170599994652}
smooth3: {arms: b, frame_left: base_footprint, frame_right: base_footprint, grip_left: true,
  grip_right: true, head_pan: 0.0, head_tilt: 0.0, pitch_left: 0.65886863377625038,
  pitch_right: 0.9701322374706598, roll_left: -0.097416192630073561, roll_right: -0.090995142984939575,
  set_head_angle: false, set_torso_height: false, torso_height: 0.0, x_left: 0.41564670220917127,
  x_right: 0.42950927932114935, y_left: 0.11440165837499308, y_right: -0.092488950087606325,
  yaw_left: -1.4049490574072303, yaw_right: 1.7289152757333341, z_left: 0.79023829791985444,
  z_right: 0.78516290821474066}
